instant-booking/README.md

```diff
@@ -0,0 +1,144 @@
+This transaction process allows your customers to insantly book listings, bypassing the default preauthorization step where the provider needs to either accept or decline the booking. This process also supports push payment methods through the request-push-payment transition. Note that the instructions below do not enable push payments in your client application.
+
+For an instant booking transaction, you completely remove the preauthorized state. The confirm-payment transition goes directly to the accepted state. 
+
+## 1. Push this process into your Flex marketplace using Flex CLI
+
+You can see the instructions in [this tutorial article](https://www.sharetribe.com/docs/tutorial/create-transaction-process/). Once you have pushed the process into your marketplace, you will be able to see it in [Flex Console > Build > Transaction processes](https://flex-console.sharetribe.com/transaction-processes/).
+
+## 2. Configure the template to use this new process
+
+There are a number of configurations in the template that are necessary for taking the new process into use.
+
+
+1. Add a new listing type in [src/config/configListing.js](https://github.com/sharetribe/web-template/blob/main/src/config/configListing.js), and add this process as the transaction type:
+
+    ```js
+    {
+        listingType: 'instant-booking',
+        label: 'Instant booking',
+        transactionType: {
+        process: 'instant-booking',
+        alias: 'release-1',
```

Comment:
```suggestion
        alias: 'instant-booking/release-1',
```
Listing type aliases currently use the full string, cf https://github.com/sharetribe/web-template/blob/main/src/config/configListing.js#L231